test/fixtures_dynamodb_test.go

```diff
@@ -59,12 +106,42 @@ func (s FixturesDynamoDbSuite) TestDynamoDb() {
 		TableName: aws.String("gosoline-test-integration-test-test-application-testModel"),
 	})
 
+	err = loader.Load(dynamoDbEnabledPurgeFixtures())
```

Comment:
this should happen after the assert of `gio`? even if it works the readability is strange because you are already loading new fixtures even if you have not asserted that the previous ones are loaded correctly?

Reply:
you are perfectly right! Will fix it